lib/byebug/helpers/toggle.rb

```diff
@@ -26,14 +24,22 @@ def enable_disable_breakpoints(is_enable, args)
           selected_breakpoints = all_breakpoints.select do |b|
             selected_ids.include?(b.id)
           end
+          return selected_breakpoints
```

Comment:
I'm taking this as a confirmation that you no longer want to work on this PR, even when you get time. I'm asking you to change those things yourselve, not asking you for permission to change them.